cmd/start.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"github.com/NodeFactoryIo/vedran/internal/router"
+	"github.com/spf13/cobra"
+	"log"
+	"net/http"
+)
+
+var (
+	shellCmd string
+)
+
+var startCmd = &cobra.Command{
+	Use:   "start",
+	Short: "Starts vedran load balancer",
+	Run:   startCommand,
+}
+
+func init() {
+	startCmd.Flags().StringVar(&shellCmd, "command", "flag", "example flag")
+	RootCmd.AddCommand(startCmd)
+}
+
+func startCommand(_ *cobra.Command, _ []string) {
+	log.Println("Starting server on :4000...")
```

Comment:
```suggestion
	log.Println("Started vedran load balancer on port 4000...")
```
and put after it's started

Reply:
I could put this log somewhere inside router initialization so I confirm that the router is initialized before I log this message, but I can't put it after this initialization line as it would log only after the server is killed (error is returned). I expanded the original message but I think the easiest solution is to keep it before this `http.ListenAndServe` call.